config/initializers/country_config.rb

```diff
@@ -83,9 +83,7 @@ class CountryConfig
   }.with_indifferent_access.freeze
 
   class << self
-    include Memery
-
-    memoize def dhis2_data_elements
+    def dhis2_data_elements
       YAML.load_file(ENV.fetch("DHIS2_DATA_ELEMENTS_FILE")).with_indifferent_access
```

Comment:
Are we okay taking a performance penalty here? Looking at the [memery docs](https://github.com/tycooon/memery), the other alternatives are:
* Create a conditional memorization condition, by setting an `:environment` accessor in country config, and disabling memoization for `test` environments
* In all the exporter specs, pass an empty block to `CountryConfig.dhis2_data_elements` to force a reload.
* Explicitly call `CountryConfig.singleton_class.clear_memery_cache!`

I prefer sidestepping the caching complications entirely, at first glance it doesn't look like our data elements file would be large enough to cause problems.

I have decided